src/libtego/CMakeLists.txt

```diff
@@ -88,17 +88,7 @@ if (APPLE)
         REQUIRED)
 endif ()
 
-# protobuf configuration
-set(Protobuf_USE_STATIC_LIBS OFF CACHE BOOL "Use statically-linked protobuf")
-set(Protobuf_ROOT_DIR "" CACHE STRING "Path to custom Protobuf installation")
-if (NOT ${Protobuf_ROOT_DIR} STREQUAL "")
-    # CMake doesn't allow us to specify a protobuf installation like with
-    # openssl. Instead, use CMAKE_PREFIX_PATH to tell CMake to check that dir
-    # first for a protobuf install
-    list(APPEND CMAKE_PREFIX_PATH ${Protobuf_ROOT_DIR})
-endif ()
-
-include(FindProtobuf)
+find_package(Protobuf CONFIG REQUIRED)
```

Comment:
Does this find_package require a particular CMake version or Protobuf version?

Reply:
aah, yepp, that was the catch...

this requires a "new" version of protobuf, but i dont know what version exactly

this may be a a problem on "always outdated" distros like debian...

Reply:
Yeah for protobuf we're stuck on the 3.21.X series unless we want to add the abseil dependency

Reply:
> unless we want to add the abseil dependency

why not?

for me this [just works](https://github.com/milahu/nur-packages/blob/master/pkgs/ricochet-refresh/default.nix)

```
$ nix why-depends nixpkgs#protobuf nixpkgs#abseil-cpp
/nix/store/kdff9wgmdncv2bhik6mppra5nni10aa8-protobuf-24.4
└───/nix/store/xpxz880b33kilbw5lccvrqjk40i05ykp-abseil-cpp-20230125.3
```

Reply:
Because in general i don't want to add new dependencies without a good reason, an specifically here I would also need to update https://github.com/blueprint-freespeech/ricochet-build